### ISSUE TITLE
support running on read-only filesystems

### ISSUE THREAD

**Issue by chuckadams** [aspirepress/AspireSync#23]
AspireSync should be able to be deployed as a Lambda function.  This won't be our preferred deployment, but it should work.  This will also mean not keeping persistent state in sqlite, so this depends on #22 as well.

**Comment by chuckadams** [comment]
Calling this one done: I'll put php-cs-fixer and thecodingmachine/phpstan-safe-rule in a follow-up PR.
